src/components/ui/FiltersPart.tsx

```
@@ -0,0 +1,194 @@
+    {label}<span className="w-3 h-3 text-blue-600">×</span>
+  </span>
+);
+
+const Checkbox = ({ label, checked, onChange }: { label: string; checked: boolean; onChange: () => void }) => (
+  <label className="flex items-center gap-2 cursor-pointer">
+    <input type="checkbox" checked={checked} onChange={onChange} className="w-4 h-4 text-blue-600 border-gray-300 rounded" />
+    <span className="text-sm text-gray-700">{label}</span>
+  </label>
+);
+
+const SearchIcon = () => <svg className="h-4 w-4 text-gray-500" fill="none" stroke="currentColor" viewBox="0 0 24 24"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" /></svg>;
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)